requirements/README.txt

```diff
@@ -0,0 +1,11 @@
+# YOU MAY DELETE THIS FILE AFTER SETTING UP DEPENDENCIES!
```

Comment:
this file needs to be removed.   It is probably a good idea if you review your own PRs to help get into the habit.  In other words, after making a PR, go to GH and click on the PR, then click on "files changed" and go through the changes in green and red and make sure everything is as you want it.  You don't have to leave review comments, but can add new commits and push. If you are quicker than me in reviewing then, it will save me some time because I will just see the most up to date version!